docs/BR.md

```diff
@@ -1595,10 +1589,22 @@ The issuing CA SHALL document in its Certificate Policy or Certification Practic
 
 ## 7.3 OCSP profile
 
+1. All OCSP responses MUST conform with RFC6960 and be the profile specified in RFC5019.
+2. Responses MUST be pre-produced as specified in RFC5019.
```

Comment:
@vanbroup So, I think the goal here was to capture the profile in RFC5019, which defined pre-produced responses (and with time) in order to establish a profile of thisUpdate, nextUpdate, producedAt, and the related transport profile (for caching directives).

It's also based on a security concern - if you have a live responder, then you have a system that isn't airgapped, and capable of producing problematic (signed) responses. This is true whether it's your issuing CA (dear God I hope not) or a delegated OCSP responder. My hope of requiring explicit pre-production was to try to move to a system where there's a one-way flow from the signing system to the serving system, such that the serving system has no access to key material and thus 'compromise' is limited to Denial of Service.

Do you have any thoughts on how to achieve those security goals, perhaps with Delegated Responder Certificates? Do you think that's an unreasonable concern?